sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows.
+
```

Comment:
I don't think it is necessary to call this out in the identity changelog. The use-case here wouldn't impact the customer perf, since it was only happening in the error path of `ThrowIfNotSafeCmdLineInput`. That isn't a case where there would be a scalability issue causing high CPU usage.

Reply:
I was thinking about that, but the change needs to be described. The customers want to know what they are getting when they are upgrading. Potentially, this still may lead to higher CPU usage, so I see no better description.

Reply:
And it is not in the error path, it is a part of normal flow - when creating credential and during each call of `GetToken()`.

Reply:
I get that `GetToken()` might be transitively impacted (by implementation detail changes in core), but the only change that impacted **identity,** is the following which is in the error path:
`ThrowIfNotSafeCmdLineInput`